src/components/GiveProject/ProjectCard.tsx

```diff
@@ -396,16 +396,15 @@ export default function ProjectCard({ project, mode }: ProjectDetailsProps) {
                   {renderGoalCompletion()}
                 </Grid>
                 <Grid item xs={6} sm={12} md={6} className="give-button-grid">
-                  <Button
+                  <PrimaryButton
                     variant="contained"
```

Comment:
please pay attention to props across your PR. variants, and in most cases props are no longer needed when moving from the mui button component to our component-library component.